content/posts/controller-exception-handling.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Controller Exception Handling"
+preview: "Custom exception handling at the controller level"
+status: "Learn how to render custom error responses"
+date: 2019-01-27T15:33:41+01:00
+draft: true
+gems: ["hanami", "hanami-controller"]
+tags: ["controller", "api"]
+versions: ["1.0"]
+---
+
+Hanami automatically handles exceptions for you. The default behaviour in production is to catch all exceptions and render an error HTML page. In the development and test environment exceptions are not caught by the framework's exception handling, instead you'll be presented with a stack trace for debugging.
+This behaviour can be changed in the `apps/[your_app_name]/application.rb` file, just look for `handle_exception` in the environment specific configuration blocks.
```

Comment:
@kaikuchn This is `handle_exceptions` (notice the plural).

From here, it isn't clear what to do. It's a commented line. Should I uncomment? Should I leave `true`, or use `false`? 😉